src/components/parts/_legacy/publishing-calendar/PublishingCalendarEntryPage.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { classNames } from 'utils/classnames';
+import { PublishingCalendarEntryProps } from 'types/content-props/publishing-calendar-props';
+import PublishingCalendarEntry from './PublishingCalendarEntry';
+import { Table } from '@navikt/ds-react';
+
+import style from './PublishingCalendar.module.scss';
+
+// For preview only - hacks layout from PublishingCalendar
+const PublishingCalendarEntryPage = (props: PublishingCalendarEntryProps) => {
```

Comment:
Kan vi legge til en sjekk her slik at den kun vises fra editoren, og 404 ellers? Se [GlobalValuesPage](https://github.com/navikt/nav-enonicxp-frontend/blob/ccc520de3bb43703981c2c18e05a6a2e8f9994fc/src/components/pages/global-values-page/GlobalValuesPage.tsx#L114) for eksempel.